src/libraries/System.ServiceModel.Syndication/src/System.ServiceModel.Syndication.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <Nullable>disable</Nullable>
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
Just curious why this needed when nullable disabled above

Reply:
I see, thanks, I thought that is the case for using `annotation` instead `disabled`, apparently not

Reply:
It is... except annotation not only allows `?`, it also persists the annotations into the metadata, meaning if you use annotations with `?` on public types, you're basically making contractual guarantees because the `?` are being recorded even though they're not validated with warnings.

Reply:
Good to know, thanks!